staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/register.go

```diff
@@ -63,7 +63,12 @@ func addToGroupVersion(scheme *runtime.Scheme, groupVersion schema.GroupVersion)
 	)
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&ListOptions{},
+		&metav1.GetOptions{},
```

Comment:
got it, so this is just an optimization because Get/Export/DeleteOption are (structurally) the same types internally and in v1, but ListOptions are not (due to the selector parsing). A comment wouldn't harm here.